CollapsibleExample/ExampleSubclassFilterViewController.m

```diff
@@ -33,44 +33,21 @@ - (void)didReceiveMemoryWarning {
 }
 
 - (void)createManagersAndPopulateData{
```

Comment:
This is minor especially because this is just an example subclass but ill bring it up so you know.

Is this a public method? Normally if you have a method like this with a name like this one you would keep it private and call it from your init method on one of your init convenience methods (- (instancetype)initWith...).